modules/balana-core/src/main/java/org/wso2/balana/xacml3/Attributes.java

```diff
@@ -210,9 +210,9 @@ public void encode(StringBuilder builder) {
         builder.append("<Attributes Category=\"").append(category.toString()).append("\">");
 
         for(Attribute attribute : attributes){
-            if(attribute.isIncludeInResult()){
-                attribute.encode(builder);
-            }
+            //if(attribute.isIncludeInResult()){
```

Comment:
Never mind. It is checked when adding attributes to the Result. It's line number 262 in org.wso2.balana.ctx.xacml3.Result.java